packages/dev/core/src/AudioV2/abstractAudio/streamingSoundInstance.ts

```diff
@@ -0,0 +1,34 @@
+import { Observable } from "../../Misc/observable";
+import { _AbstractSoundInstance } from "./abstractSoundInstance";
+import type { IStreamingSoundOptions, StreamingSound } from "./streamingSound";
+
+/** @internal */
+export abstract class _StreamingSoundInstance extends _AbstractSoundInstance {
```

Comment:
Is it possible for sound instances to fail to load? There doesn't seem to be any code that handles errors.

Reply:
I took a stab at it in commit https://github.com/BabylonJS/Babylon.js/pull/15839/commits/ffbce508d743cad1097c14e00f723fa0b7d596b3. If that looks right then I'll go ahead and do something similar for the other classes.

Reply:
Discussed offline.